src/ralph/ssl_certificates/management/commands/update_service_env.py

```diff
@@ -0,0 +1,58 @@
+import sys
+
+from django.core.management.base import BaseCommand
+
+from ralph.assets.models.assets import ServiceEnvironment
+from ralph.dns.dnsaas import dnsaas_client
+from ralph.ssl_certificates.models import SSLCertificate
+
+
+class Command(BaseCommand):
+    help = 'Checks the compliance of services in SSL Certificates'
+
+    def update_from_record(self, results):
+        for value in results:
+            domain = value['content']
+            service_dns = value['service_name']
```

Comment:
what if DNS record doesn't have assigned service?

Reply:
Then we'll get empty string and information in log that this service ('') is not exist in ralph.